internal/sourcecode/templates/redwood/Dockerfile

```diff
@@ -39,7 +39,7 @@ COPY --from=api_build /app/api/dist /app/api/dist
 COPY --from=api_build /app/api/db /app/api/db
 COPY --from=api_build /app/node_modules/.prisma /app/node_modules/.prisma
 
-EXPOSE 8911
+EXPOSE 8910
```

Comment:
On Fly this line is not used at all, so we could get rid of it. It might be useful for other platforms, though.

Reply:
Considering it's generated from flyctl launch command, it makes sense to keep it minimal then. Removed this line.